packages/airnode-examples/src/scripts/deploy-airnode.ts

```diff
@@ -18,8 +18,8 @@ const main = async () => {
   const deployCommand = [
     `docker run -it --rm`,
     isWindows() ? '' : `-e USER_ID=$(id -u) -e GROUP_ID=$(id -g)`,
-    integrationInfo.airnodeType === 'aws' && `-v "${integrationPath}/aws.env:/app/aws.env"`,
-    integrationInfo.airnodeType === 'gcp' && `-v "${integrationPath}/gcp.json:/app/gcp.json"`,
+    integrationInfo.airnodeType === 'aws' && `-v "${integrationPath}/aws.env:/app/config/aws.env"`,
+    integrationInfo.airnodeType === 'gcp' && `-v "${integrationPath}/gcp.json:/app/config/gcp.json"`,
     `-v ${integrationPath}:/app/config`,
```

Comment:
Is this actually necessary? The whole `integrationPath` is mounted as `/app/config` so the files will be there, right?

Reply:
Yes the files will be there- good catch. I've removed the auth file mounts from the deploy and remove airnode-examples scripts.